### DISCUSSION TITLE
Welcome to NDSROM Discussions!

### DISCUSSION THREAD

**Discussion by Jenna1337** [Jenna1337/NDSROM#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rdrpenguin04** [comment]
Hello! I'm Ray, and I'm "good" at disassembly. ... That's what I intend to do.
